src/ApiPort.VisualStudio.Common/AssemblyFile.cs

```diff
@@ -12,16 +12,20 @@ namespace ApiPortVS
     internal class AssemblyFile : IAssemblyFile
     {
         private readonly string _path;
+        private readonly bool _skipBinaryIfPackageExists;
 
-        public AssemblyFile(string path)
+        public AssemblyFile(string path, bool skipBinaryIfPackageExists = false)
         {
             _path = path;
+            _skipBinaryIfPackageExists = skipBinaryIfPackageExists;
         }
 
         public string Name => _path;
 
         public bool Exists => File.Exists(_path);
 
+        public bool SkipBinaryIfPackageExists => _skipBinaryIfPackageExists;
```

Comment:
What is this supposed to do? `IAssemblyFile` is meant to be a simple file abstraction and this seems to add some more logic to it

Reply:
@twsouthwick @alinapopa 

Would a better place be to update [IApiPortOptions.cs](https://github.com/Microsoft/dotnet-apiport/blob/master/src/Microsoft.Fx.Portability/IApiPortOptions.cs#L13)?  And then modify the logic in [AnalyzeOptions.cs](https://github.com/Microsoft/dotnet-apiport/pull/465/files#diff-d16b3fc07e2e9cb3be40d03c9855a338R165) based on whether a directory was specified or not?
```
    public interface IApiPortOptions
    {
        /// <summary>
        /// Key: IAssemblyFile
        /// Value: true if the file was explicitly specified and false otherwise.
        /// </summary>
        ImmutableDictionary<IAssemblyFile, bool> InputAssemblies { get; }
```

Reply:
@conniey It sounds like a good idea to me; I added the change.